packages/helper-settings/SettingsForm.js

```diff
@@ -53,8 +58,11 @@ export default class Form extends Component {
     if (!response.login) {
       this.setState({
         tokenLoaded: false,
-        errorMessage: response.message || 'Something went wrong',
       });
+      this.tokenInputEl.setCustomValidity(
+        response.message || 'Something went wrong',
+      );
+      this.tokenInputEl.reportValidity();
```

Comment:
This solution is inspirited by https://github.com/developit/preact-mdl/blob/master/src/index.js#L873-L878 so it shouldn't be completely wrong 😉